src/domain/users-ip/index.types.d.ts

```diff
@@ -0,0 +1,15 @@
+type UserIp = {
+  readonly id: UserId
+  lastIPs: IPType[]
```

Comment:
Why is this called `lastIPs`? Is the array pruned?

Reply:
not as of now but eventually it could/should

Reply:
I prefer to use names that reflect the _is_ rather than the _should be_ but its not a big deal.